assets/js/base/components/cart-checkout/shipping-rates-control/index.tsx

```diff
@@ -12,16 +11,78 @@ import {
 	getShippingRatesRateCount,
 } from '@woocommerce/base-utils';
 import { useStoreCart } from '@woocommerce/base-context/hooks';
+import { CartResponseShippingRateItem } from '@woocommerce/type-defs/cart-response';
+import { ReactElement } from 'react';
 
 /**
  * Internal dependencies
  */
-import ShippingRatesControlPackage from '../shipping-rates-control-package';
+import ShippingRatesControlPackage, {
+	PackageRateRenderOption,
+} from '../shipping-rates-control-package';
+
+interface PackagesProps {
+	// eslint-disable-next-line camelcase
+	packages: CartResponseShippingRateItem[];
+	collapse?: boolean;
+	collapsible?: boolean;
+	showItems?: boolean;
+	noResultsMessage: ReactElement;
+	renderOption: PackageRateRenderOption;
+}
 
 /**
- * @typedef {import('react')} React
+ * Renders multiple packages within the slotfill.
+ *
+ * @param {Object} props Incoming props.
```

Comment:
Is this docblock needed now Packages is typed?

Reply:
Not _needed_ but  I don't think we should actively remove them. It being there gives more explanation of each prop so I don't see its existence as a negative thing.